lib/shopify-cli/js_deps.rb

```diff
@@ -55,43 +55,50 @@ def yarn(verbose = false)
       cmd = %w(yarn install)
       cmd << '--silent' unless verbose
 
-      CLI::Kit::System.system(*cmd, chdir: ctx.root) do |out, err|
-        ctx.puts out
-        err.lines.each do |e|
-          ctx.puts e
-        end
-      end.success?
+      run_install_command(cmd)
     end
 
     def npm(verbose = false)
       cmd = %w(npm install --no-audit --no-optional)
-      cmd << '--silent' unless verbose
+      cmd << '--quiet' unless verbose
```

Comment:
did we just do this wrong the first time? or are these different

Reply:
It depends on your goals. To align this I updated to `--quiet` to let me capture error output in case an error occurred. `--silent` stops all messaging even errors so I couldn't output anything on error.

Reply:
I just realized I read your question completely wrong. A better answer:

No, we didn't do it wrong. `--silent` silences ALL output. `--quiet` silences all output except errors. I wanted to be able to display an error in case the installation failed.